core/src/main/res/values/strings.xml

```diff
@@ -24,5 +24,8 @@
     <string name="device_view_title">Device Information</string>
     <string name="logs_view_title">Logs</string>
     <string name="file_issue_view_title">File Issue</string>
+    <string name="search_logs_hint">Search the Logs</string>
+    <string name="search_logs_button_text">Search</string>
+    <string name="placeholder_logs">Temporary text to be replaced by logs</string>
```

Comment:
```suggestion
    <string name="placeholder_logs">Search results will appear here…</string>
```

Reply:
Once I show the actual logs on the developer menu, the placeholder_logs string resource will be deleted.

Reply:
Yes!!

Consider how this text will read to a user, before that point. What is the most useful thing a user can read? The phrase "placeholder" is a UI development term for this concept. My feedback is just that the verbiage could be more user-aware.